python/pyiceberg/types.py

```diff
@@ -133,32 +176,34 @@ class FixedType(PrimitiveType):
         False
     """
 
-    __root__: str = Field()
-    _len: int = PrivateAttr()
+    root: int = Field()
```

Comment:
Why is the length called `root` now? Can we call it `len` or similar instead? I think it is confusing this way.

Reply:
This is because it serializes to a single value instead of a dict, so we have to map it onto root.